handlers/createGame.go

```diff
@@ -10,6 +10,7 @@ import (
 
 type CreateGameResponse struct {
 	GameId string `json:"game_id"`
+	Game truco.Game `json:"game"`
```

Comment:
Could this just be `Game`? I'm assuming `GameId` will be a part of the `Game` object anyway, if that's the case then no point duplicating in the response